ext/sockets/tests/gh16267.phpt

```diff
@@ -3,20 +3,20 @@ GH-16267 - overflow on socket_strerror argument
 --EXTENSIONS--
 sockets
 --SKIPIF--
-<?php if (PHP_INT_SIZE != 8) die('skip 64-bit only'); ?>
+<?php if (PHP_INT_SIZE != 8) die("skip this test is for 64bit platform only"); ?>
 --FILE--
 <?php
 try {
-	socket_strerror(PHP_INT_MIN);
+	socket_strerror(-2147483648);
 } catch (\ValueError $e) {
 	echo $e->getMessage() . PHP_EOL;
 }
 try {
-	socket_strerror(PHP_INT_MAX);
+	socket_strerror(2147483648);
 } catch (\ValueError $e) {
 	echo $e->getMessage() . PHP_EOL;
 }
 ?>
 --EXPECTF--
-socket_strerror(): Argument #1 ($error_code) must be between %s and %s
-socket_strerror(): Argument #1 ($error_code) must be between %s and %s
+socket_strerror(): Argument #1 ($error_code) must be between %s and %d
+socket_strerror(): Argument #1 ($error_code) must be between %s and %d
```

Comment:
```suggestion
socket_strerror(): Argument #1 ($error_code) must be between %d and %d
socket_strerror(): Argument #1 ($error_code) must be between %d and %d
```

Reply:
sorry does not work for me, I guess because it s a negative.

Reply:
Right, use `%i`